Project.toml

```
@@ -14,7 +14,7 @@ SolverCore = "ff4d7338-4cf1-434d-91df-b86cb86fb843"
 SolverTools = "b5612192-2639-5dc1-abfe-fbedd65fab29"
 
 [compat]
-Krylov = "0.4, 0.5, 0.6, 0.7, 0.8"
+Krylov = "~0.7.4, 0.8"
```

```suggestion
Krylov = "~0.7.4"
```
I left this by mistake.